src/docs/product/integrations/aws-lambda/index.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: AWS Lambda (Node)
+sidebar_order: 1
+description: "Learn more about Sentry's AWS Lambda integration and how you can automatically instrument your Node Lambda functions with Sentry without changing your code."
+---
+
+<Alert title="Note" level="info">
+
+Looking for instructions to set up the Node serverless SDK manually? [Check out these docs instead.](/platforms/node/guides/aws-lambda/)
+
+</Alert>
+
+Connect Sentry to your AWS account to automatically add Sentry error monitoring and transactions to your Node Lambda functions.
+
+
+## Install
+
+1. Navigate to **Settings > Integrations > AWS Lambda**.
+
+   ![](directory.png)
+
+1. Press **Add Installation** which will open the installation modal.
+
+   ![](integration_detail.png)
+
+1. Select a project to use for your Lambda functions. Note that this project cannot be changed later. If you only have a single project, this step is skipped.
+
+   ![](project.png)
+
+1. Click on the button that says **Configure AWS**.
+
+   ![](aws_account.png)
+  
+1. Scroll to the bottom of the create stack page, mark the checkbox at the bottom and press the **Create Stack** button. This will create a CloudFormation stack in the region you are currently in. This stack allows Sentry to instrument your Lambda functions automatically.
+
+   ![](cloudformation.png)
+
+1. Wait a minute or two for the stack to be created. When you see the status change to **Create Complete**, copy your AWS account number, select the region you want to use, and finally press **Next** in the installation modal.
+
+   ![](stack_created.png)
+   
+<Alert title="Note" level="info">
+
+You only need to create the CloudFormation stack once per account even when installing the integration on multiple regions. If you are setting up the integration on a second region, verify that the externalId in the form matches the one for the existing CloudFormation stack on the "Parameters" tab.
+
+</Alert>
+
+   
+1. Review the Lambda functions you want instrumented with Sentry. Lambda functions which are selected will get the Sentry layer and necessary environment variables automatically added when you press **Finish Setup**. Note that you can enable/disable Sentry on Lambda functions after installation as well.
+
+   ![](select_lambdas.png)
+
+1. Depending on how many Lambda functions you enable, you might need to wait a few minutes for this step to complete. If any functions failed to instrument, follow the troubleshooting guide below.
```

Comment:
This isn't a step, unless we simply want to say "Wait" - otherwise, I think what we mean to say is
```suggestion
Wait a few minutes as the Lambda functions are enabled. If any functions failed to instrument, follow the suggestions in Troubleshooting.
```